README.rst

```diff
@@ -66,7 +66,7 @@ Features
 Requirements
 ------------
 
-``pyuploadcare`` requires Python 2.6, 2.7, 3.3, 3.4, 3.5 or 3.6
+``pyuploadcare`` requires Python 2.7, 3.4, 3.5 or 3.6
```

Comment:
In here can we say, that `pyuploadcare` does not tested with other versions of python, but if user want - he can try to use other versions?

Reply:
yup, you missed the conversation yesterday.
I agree that we can state that it should work for quite some time with old version and offered to create an extended tox config.

but. officially we're dropping support for obsolete, so `setup.py` should reflect that.